provisioning/service/src/ProvisioningServiceClient.cs

```diff
@@ -104,6 +111,75 @@ public static ProvisioningServiceClient CreateFromConnectionString(string connec
             return new ProvisioningServiceClient(connectionString, httpTransportSettings);
         }
 
+        /// <summary>
+        /// Create a new instance of the <code>ProvisioningServiceClient</code> from the provided hostName and <code>TokenCredential</code>
```

Comment:
You can use this alias for code that will reduce line length by a reasonable amount, especially since we're using it several times.

```suggestion
        /// Create a new instance of the <c>ProvisioningServiceClient</c> from the provided hostName and <c>TokenCredential</c>
```

Reply:
Updated